pkg/registry/core/node/strategy.go

```diff
@@ -104,7 +104,7 @@ func dropDisabledFields(node *api.Node, oldNode *api.Node) {
 	}
 
 	if !utilfeature.DefaultFeatureGate.Enabled(features.RecursiveReadOnlyMounts) {
```

Comment:
It seems weird that this is tied to the RecursiveReadOnlyMounts since it's also used by UserNamespaces.

Reply:
The flag for UserNamespaces is still not included in the Kubernetes core API struct